cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -2323,6 +2341,13 @@ private List<Field<?>> getLatestApplicationKeys(String namespaceId, String appNa
     return fields;
   }
 
+  private List<Field<?>> getApplicationKeys(String namespaceId, String appName) {
```

Comment:
this looks like the same thing as the getNamespaceApplicationKeys() method below